### PR TITLE
docs(tr): Add Turkish language support

### PR DESCRIPTION
**Pull request by ImBIOS**
## Summary

Part of:

- #1222 

## Test Plan

It should be fundamentally correct, but still open for suggestion.

### CODE REVIEW

**Comment by ematipico** [comment]
Hey @ImBIOS, I noticed you opened a bunch of PRs for translating content in other languages:
- do you speak all those languages?
- do you know someone that is going to help us with all these translations?

**Comment by ImBIOS** [comment]
@ematipico I don't daily speak all those languages, I just love to learn multiple languages, but I'm committed to ensuring those translations are accurate and high-quality. I've already reached out to some native speakers who can assist with these translations. My goal is to make our content accessible to a wider audience, and I believe it's a valuable effort.

I'm coordinating with these individuals to ensure that the translations are not just linguistically correct, but also culturally relevant and technically accurate. I'm taking this seriously to maintain the integrity and quality of the content.

File: packages/@biomejs/biome/README.tr.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+[vscode-badge]: https://badgen.net/vs-marketplace/v/biomejs.biome?label=vscode&icon=visualstudio&color=green
+[vscode-url]: https://marketplace.visualstudio.com/items?itemName=biomejs.biome
+[open-vsx-badge]: https://badgen.net/open-vsx/version/biomejs/biome?label=open-vsx&color=green
+[open-vsx-url]: https://open-vsx.org/extension/biomejs/biome
+
+</div>
+
+**Biome**, web projeleri için performanslı bir alet takımıdır, söz konusu projelerin sağlığını korumak için geliştirici araçları sağlamayı amaçlar.
+
+**Biome, _JavaScript_, _TypeScript_, _JSX_ ve _JSON_ için [hızlı bir formatlayıcıdır](./benchmark#formatting)** ve **[_Prettier_ ile %96 uyumluluk](https://console.algora.io/challenges/prettier)** puanına sahiptir.
```
Comment (anonrig) [inline packages/@biomejs/biome/README.tr.md:30]:
```suggestion
**Biome, _JavaScript_, _TypeScript_, _JSX_ ve _JSON_ için [hızlı bir formatlayıcıdır](./benchmark#formatting)** ve **[_Prettier_ ile %96 uyumludur](https://console.algora.io/challenges/prettier)**.
```

File: packages/@biomejs/biome/README.tr.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+**Biome, _JavaScript_, _TypeScript_ ve _JSX_ için [performanslı bir lintleyicidir](https://github.com/biomejs/biome/tree/main/benchmark#linting)** ve ESLint, TypeScript ESLint ve [diğer kaynaklardan](https://github.com/biomejs/biome/discussions/3) **170'den fazla kural** içerir.
```
Comment (anonrig) [inline packages/@biomejs/biome/README.tr.md:32]:
```suggestion
**Biome, _JavaScript_, _TypeScript_ ve _JSX_ için [performanslı bir derleyicidir](https://github.com/biomejs/biome/tree/main/benchmark#linting)** ve ESLint, TypeScript ESLint ve [diğer kaynaklardan](https://github.com/biomejs/biome/discussions/3) **170'den fazla kural** içerir.
```

File: packages/@biomejs/biome/README.tr.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+Kodunuzu geliştirmenize ve daha iyi bir programcı olmanıza yardımcı olacak **ayrıntılı ve bağlamsal tanılar sunar**!
+
+**Biome**, [bir editör içinde etkileşimli olarak kullanılmak](https://biomejs.dev/guides/integrate-in-editor/) üzere baştan tasarlanmıştır.
+Yazarken hatalı kodu formatlayabilir ve lintleyebilir.
```
Comment (anonrig) [inline packages/@biomejs/biome/README.tr.md:36]:
```suggestion
Yazarken hatalı kodu formatlayabilir ve lintleyebilirsiniz.
```

File: packages/@biomejs/biome/README.tr.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+npx @biomejs/biome lint ./src
+
+# format, lint vb. çalıştır ve güvenli önerileri uygula
+npx @biomejs/biome check --apply ./src
+
+# CI ortamlarında tüm dosyaları format, lint vb. ile kontrol et
+npx @biomejs/biome ci ./src
+```
+
+Biome'u yüklemek istemeden denemek istiyorsanız, WebAssembly'ye derlenmiş [çevrimiçi oyun alanını](https://biomejs.dev/playground/) kullanın.
```
Comment (anonrig) [inline packages/@biomejs/biome/README.tr.md:60]:
```suggestion
Biome'u yüklemek istemeden denemek istiyorsanız, WebAssembly'ye derlenmiş [oyun alanını](https://biomejs.dev/playground/) kullanabilirsiniz.
```

File: packages/@biomejs/biome/README.tr.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+## Dokümantasyon
+
+Biome hakkında daha fazla bilgi edinmek için [ana sayfamızı][biomejs] ziyaret edin,
+veya Biome'u kullanmaya başlamak için doğrudan [Başlangıç Rehberi'ne][getting-started] gidin.
+
+## Biome H
+
+akkında Daha Fazlası
```
Comment (anonrig) [inline packages/@biomejs/biome/README.tr.md:69]:
```suggestion
## Biome Hakkında Daha Fazlası
```

File: packages/@biomejs/biome/README.tr.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+**Biome**, sağlam varsayılanlara sahiptir ve yapılandırma gerektirmez.
+
+**Biome**, modern web geliştirmenin [tüm ana dillerini][language-support] desteklemeyi amaçlar.
```
Comment (anonrig) [inline packages/@biomejs/biome/README.tr.md:73]:
```suggestion
**Biome**, modern web geliştirmenin [tüm dillerini][language-support] desteklemeyi amaçlar.
```

File: website/astro.config.ts [hunk 7]
```diff
@@ -97,6 +112,7 @@ export default defineConfig({
 								ja: "はじめる",
 								"zh-CN": "入门",
 								"pt-BR": "Primeiros passos",
+								tr: "Başlarken",
```
Comment (anonrig) [inline website/astro.config.ts:115]:
```suggestion
								tr: "Başla",
```

File: website/astro.config.ts [hunk 8]
```diff
@@ -106,6 +122,7 @@ export default defineConfig({
 								ja: "手動インストール",
 								"zh-CN": "手动安装",
 								"pt-BR": "Instalação manual",
+								tr: "El ile kurulum",
```
Comment (anonrig) [inline website/astro.config.ts:125]:
```suggestion
								tr: "Kurulum kılavuzu",
```

File: website/astro.config.ts [hunk 9]
```diff
@@ -115,6 +132,7 @@ export default defineConfig({
 								ja: "大きなプロジェクトでのBiomeの使用方法",
 								"zh-CN": "大型项目使用 Biome",
 								"pt-BR": "Usando o Biome em projetos grandes",
+								tr: "Büyük projelerde Biome kullanma",
```
Comment (anonrig) [inline website/astro.config.ts:135]:
```suggestion
								tr: "Büyük projelerde Biome",
```

File: website/astro.config.ts [hunk 10]
```diff
@@ -133,6 +152,7 @@ export default defineConfig({
 								ja: "Biome をあなたのエディタに導入する",
 								"zh-CN": "编辑器中使用 Biome",
 								"pt-BR": "Integrando o Biome no seu editor",
+								tr: "Biome'u editörünüze entegre edin",
```
Comment (anonrig) [inline website/astro.config.ts:155]:
```suggestion
								tr: "Biome'u editörünüze ekleyin",
```

File: website/astro.config.ts [hunk 11]
```diff
@@ -152,6 +173,7 @@ export default defineConfig({
 								ja: "Analyzer",
 								"zh-CN": "分析器",
 								"pt-BR": "Analisador",
+								tr: "Analizör",
```
Comment (anonrig) [inline website/astro.config.ts:176]:
```suggestion
								tr: "İnceleyici",
```

File: website/astro.config.ts [hunk 12]
```diff
@@ -173,6 +196,7 @@ export default defineConfig({
 										ja: "Prettier との違い",
 										"zh-CN": "与 Prettier 的区别",
 										"pt-BR": "Diferenças em relação ao Prettier",
+										tr: "Prettier ile farklılıklar",
```
Comment (anonrig) [inline website/astro.config.ts:199]:
```suggestion
										tr: "Prettier arasındaki farklar",
```

File: website/astro.config.ts [hunk 13]
```diff
@@ -182,12 +206,14 @@ export default defineConfig({
 										ja: "Formatterオプションに対する考え方",
 										"zh-CN": "格式化配置理念",
 										"pt-BR": "Princípios de configuração",
+										tr: "Biçimlendirici seçenek felsefesi",
```
Comment (anonrig) [inline website/astro.config.ts:209]:
```suggestion
										tr: "Biçimlendirici prensipleri",
```

File: website/astro.config.ts [hunk 14]
```diff
@@ -211,6 +238,7 @@ export default defineConfig({
 						ja: "リファレンス",
 						"zh-CN": "参考",
 						"pt-BR": "Referências",
+						tr: "Referans",
```
Comment (anonrig) [inline website/astro.config.ts:241]:
```suggestion
						tr: "Referanslar",
```

File: website/astro.config.ts [hunk 15]
```diff
@@ -233,13 +262,19 @@ export default defineConfig({
 								ja: "VSCode拡張機能",
 								"zh-CN": "VSCode 扩展",
 								"pt-BR": "Extensão do VSCode",
+								tr: "VSCode uzantısı",
```
Comment (anonrig) [inline website/astro.config.ts:265]:
```suggestion
								tr: "VSCode eklentisi",
```

**Review by anonrig — CHANGES_REQUESTED** [review]

**Comment by anonrig** [comment]
> @ematipico I don't daily speak all those languages, I just love to learn multiple languages, but I'm committed to ensuring those translations are accurate and high-quality. I've already reached out to some native speakers who can assist with these translations. My goal is to make our content accessible to a wider audience, and I believe it's a valuable effort.
> 
> I'm coordinating with these individuals to ensure that the translations are not just linguistically correct, but also culturally relevant and technically accurate. I'm taking this seriously to maintain the integrity and quality of the content.

I gave up after a while but they are not exact translations. It feels "artificial" when you read it. (PS: I'm Turkish and speak/write Turkish fluently)

**Comment by ImBIOS** [comment]
@anonrig Thanks for your review, it's a great experience for me to learn along the way.

**Comment by ematipico** [comment]
> > @ematipico I don't daily speak all those languages, I just love to learn multiple languages, but I'm committed to ensuring those translations are accurate and high-quality. I've already reached out to some native speakers who can assist with these translations. My goal is to make our content accessible to a wider audience, and I believe it's a valuable effort.
> > I'm coordinating with these individuals to ensure that the translations are not just linguistically correct, but also culturally relevant and technically accurate. I'm taking this seriously to maintain the integrity and quality of the content.
> 
> I gave up after a while but they are not exact translations. It feels "artificial" when you read it. (PS: I'm Turkish and speak/write Turkish fluently)

Yeah, that's what I thought at the beginning, I think all these translations are translated using Google Translate and/or some AI tool.

Sorry @anonrig taking so much of your time. @ImBIOS, this is not the approach we want to take for translating our documentation. Unless you speak those languages, we don't want to use automated tools, because the majority of the work falls on the reviewer, not the creator.

**Pull request closed without merging**